### ISSUE TITLE
Tumbling E chart

### ISSUE THREAD

**Issue by gtwy** [sightline-ophthalmic/OS-Acuity#1]
Feature request to add a "tumbling E" chart for non-verbal patients or patients who don't know our alphabet. It is a chart that is made entirely of Es, each rotated in a random orientation.
 
![Medical_Instrument_Eye_Visual_acuity_chart_E_chart](https://user-images.githubusercontent.com/11441407/58514784-df0cf780-8170-11e9-96af-aa3a5d4e12cb.jpg)

**Comment by browerbr** [comment]
Excelent! Numbers (1,2,3,4,5) also help.

I use an another free system for visual chart, using the same idea (compute stick + monitor + bluetooth controller), but i noted that there are some options with this method (open source or free in windows), and pc sticks with windows are somewhat expensive.

Would be a nice idea to create something that would be installed on android sticks (so much cheaper) and excelent for this kind of work (many commercials system use android pc stick + IR or bluetooth controllers).

**Comment by gtwy** [comment]
It probably does work fine on other operating systems. But you'll need to make sure to disable any type of DPI scaling and do some test measurements for yourself. The solution we found is to use Intel Compute Sticks for around $150 a stick. It's significantly cheaper than the $4,000 per exam room that closed source solutions cost.

You can [edit the source](https://github.com/sightline-ophthalmic/OS-Acuity/blob/ec4f824ce2a1fd0862a97a938607b21f16ef8275/wwwdata/optotype.js#L147) to include any letters or numbers you want to see on screen. Please note that after adding more letters or numbers, you may need to add [additional blacklist words](https://github.com/sightline-ophthalmic/OS-Acuity/blob/ec4f824ce2a1fd0862a97a938607b21f16ef8275/wwwdata/optotype.js#L177).

**Comment by browerbr** [comment]
> It probably does work fine on other operating systems. But you'll need to make sure to disable any type of DPI scaling and do some test measurements for yourself. The solution we found is to use Intel Compute Sticks for around $150 a stick. It's significantly cheaper than the $4,000 per exam room that closed source solutions cost.
> 
> You can [edit the source](https://github.com/sightline-ophthalmic/OS-Acuity/blob/ec4f824ce2a1fd0862a97a938607b21f16ef8275/wwwdata/optotype.js#L147) to include any letters or numbers you want to see on screen. Please note that after adding more letters or numbers, you may need to add [additional blacklist words](https://github.com/sightline-ophthalmic/OS-Acuity/blob/ec4f824ce2a1fd0862a97a938607b21f16ef8275/wwwdata/optotype.js#L177).

Agree. Your system is very nice. I suggested the android version because we can buy android stick devices with android 8.1 for about $50 and that work faster that Intel Compute Sticks for $150.
So, a cheaper and better solution. I will try to find some programmer that can port your system to .apk and test and post here.  Thank you very much.
